DualityEditorPlugins/EditorBase/PropertyEditors/Resources/FontContentPropertyEditor.cs

```diff
@@ -27,7 +27,7 @@ protected override void OnGetValue()
 			Font font = this.GetValue().FirstOrDefault() as Font;
 
 			bool lastCanRenderFont = this.canRenderFont;
-			this.canRenderFont = (font != null && font.EmbeddedTrueTypeFont != null);
+			this.canRenderFont = (font != null && font.GetGlyphBitmap(' ') != null);

```

Comment:
This line should remain unchanged as well, because we can, in fact, not render the font, if there is no embedded truetype font. Stating otherwise would be an error. 

(We might need to figure something out in order to allow the user to still access the kerning field non-readonly, but one thing at a time. That's probably for another PR.)